lib/card_generator.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+require './lib/card'
+
+class CardGenerator
+    attr_reader :file, :cards
```

Comment:
In some classes, you include a space after your `attr_reader` line, and in some, you don't. I'd pick a style and stick with it. It is convention to add a blank line after the `attr_reader` line.

I'm not sure we need an `attr_reader` for `@file` since it doesn't say in the interaction pattern that this attribute will ever be called from outside this class.